README.md

```diff
@@ -200,7 +200,7 @@ require('babel-core').transform('code', {
 })
 ```
 
-## Use with [`babel-macros`][babel-macros]
+## Use with [`babel-macros`](https://github.com/kentcdodds/babel-macros)
```

Comment:
Thanks! The `babel-macros` link is actually at the bottom of the README file (with the rest of the links). Looks like there was just a typo there, could you change this back and just fix the typo? Thanks!

Reply:
Ah thats cool, didnt know that you could alias links in markdown like this 😃